docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
+==================
+
+Hello hello, fellow documentarians! It's that time of year again: We’re very
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **May 6-8, 2018, in Portland, Oregon**.
```

Comment:
North American > Portland?

Reply:
As a description seems fine, esp with Portland in the bold text.